notes/style-guide.md

```diff
@@ -4,31 +4,44 @@ Style guide for the standard library
 This is very much a work-in-progress and is not exhaustive.
 
 ## File structure
+Agda programs are structured in modules. The first module in each file of the library is the top-level module whose name always matches the filename.
```

Comment:
This seems to be describing Agda in general rather than the style in which the standard library is written? It's therefore maybe a little off topic?

Reply:
Well, i was trying to make explicit for the potential contributors the conventions on file naming. I wanted here also to say something about how the library is structured, but i wasn't sure.

Reply:
> Well, i was trying to make explicit for the potential contributors the conventions on file naming. I wanted here also to say something about how the library is structured, but i wasn't sure.

I think the place for talking about the structure of the library is the README folder, not the style guide.

Reply:
OK, i'll remove this